### DISCUSSION TITLE
Are there automated ways to garbage collect old entries in the runs table?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17640] [Q&A]
Hi team, we are having performance issues with the runs table (too entries and we use postgres), are there any non manual ways to keep this table trimmed to a certain number of days? Thanks

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698941718594439?thread_ts=1698941718.594439&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment]
Check out this discussion here about removing old entries from the DB https://github.com/dagster-io/dagster/discussions/12047. You could potentially make a job that runs this operation automatically and put it on a schedule.
